src/lib/locationUtils.js

```diff
@@ -25,6 +25,18 @@ export function isWithinRegion(test, region) {
   )
 }
 
+export function getBounds({start, end}) {
+  if (!start || !end) {return []}
```

Comment:
Possibly, but we don't have that need right now. I think I'd rather just have the api be that you pass in both bounds, and the function computes as such. to get a one element array you can always just pass start, start.